AWSSDKSwiftCLI/Sources/SPR/Models/Describe/Describe.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct Describe: Codable {
```

Comment:
The many Codable structures below are used to read & write the standard JSON formatted data for registry version lists and version metadata.